tokenLists/fantom.mainnet.tokenlist.json

```diff
@@ -421,7 +413,6 @@
             "_scan": "https://ftmscan.com/token/0x47d0d625638b56084e76b8720475d175d171af9A",
             "chainId": 250,
             "decimals": 18,
```

Comment:
add logo

Reply:
The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it